changelogs/fragments/parseable_locale.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - added new function to module utils to choose best possible locale.
```

Comment:
Would it possible to add test for this change?

Reply:
once we agree this is something we want, i'll add unit tests